src/flow/journey.js

```diff
@@ -30,11 +30,7 @@ const options = userOpts => {
   if (typeof userOpts.session === 'function') {
     sessionProvider = userOpts.session;
   } else {
-    const cookie = Object.assign(
-      { domain: parseUrl(userOpts.baseUrl).hostname },
-      userOpts.session.cookie || {}
-    );
-    const sessionOpts = Object.assign({}, userOpts.session, { cookie });
+    const sessionOpts = Object.assign({}, userOpts.session, { cookie: {} });
```

Comment:
This will prevent anyone from applying their own cookie options if they needed to

Reply:
yea good point ill sort this

Reply:
@bendiggle please re-review